src/content/docs/synthetics/synthetic-monitoring/using-monitors/device-emulation.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Device emulation
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Using monitors
+translate:
+  - jp
+metaDescription: Emulating mobile or tablet devices in browser based synthetic monitors
+redirects:
+---
+
+Device emulation is an optional setting available for simple and scripted browser monitors. This setting simulates the viewport and capabilities of a mobile or tablet device using the emulation capabilities of Chrome DevTools. 
+
+Device emulation allows you to test a browser based application using a simulated generic mobile or tablet device. This setting does not change the network, user agent, or device type being used to execute your synthetic monitor. Device emulation does not allow you to install and test mobile applications.
+
+## Device descriptions [#devices]
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Device type
+      </th>
+
+      <th>
+        Device details
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        Mobile
+      </td>
+
+      <td>
+        * Resolution (Portriat): 400x800
+        * Resolution (Landscape): 800x400
+        * Pixel Ratio: 3
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Tablet
+      </td>
+
+      <td>
+        * Resolution (Portriat): 768x1024
+        * Resolution (Landscape): 1024x768
+        * Pixel Ratio: 2
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Requirements [#requirements]
+
+* Chrome 100+ runtimes only
+* Private locations must use [Job Manager](docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide/) and version 1.1.96 of synthetics-node-browser-runtime or higher
```

Comment:
@bpeckNR is this page supposed to have the settings that you refer to in this page: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx ?

If so, could you add any details here about how you make configurations for device emulations? Maybe this is easier that I imagine, but is there a screen where users selected options or something?